internal/action/clone.go

```diff
@@ -120,7 +120,7 @@ func (s *Action) clone(ctx context.Context, repo, mount, path string) error {
 
 	// make sure the parent directory exists.
 	if parentPath := filepath.Dir(path); !fsutil.IsDir(parentPath) {
-		if err := os.MkdirAll(parentPath, 0700); err != nil {
+		if err := os.MkdirAll(parentPath, 0o700); err != nil {
```

Comment:
I'd argue that `0700` is already octal and the way we are used to see permission in the unix world.

Reply:
This is a [gofumpt](https://github.com/mvdan/gofumpt) change. I don't have a strong opinion on this, but since we don't have strong community of "style reviewers" here I'd prefer to rely on automation as much as possible, i.e. follow the conventions that the tools suggest.